src/Interface/Decidable/Instance.agda

```diff
@@ -15,8 +15,6 @@ open import Relation.Binary renaming (Decidable to Decidable²)
 open import Relation.Binary.PropositionalEquality
 open import Relation.Nullary
```

Comment:
From `src/Interface/Decidable/Instance.agda` we can remove the following:

```
open import Data.List
open import Data.List.Properties
open import Relation.Nullary
```

As well as the variable `P` in the `private variable` declarations.